charts/operator-wandb/charts/app/templates/_deployment.tpl

```diff
@@ -208,6 +208,26 @@ spec:
               value: "0.0.0.0"
             {{- end }}
             {{- end }}
+            - name: ACCESS_KEY
+              {{- if .Values.global.bucket.accessKey }}
+              value: "{{ .Values.global.bucket.accessKey | default .Values.global.defaultBucket.accessKey }}"
+              {{- else }}
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ include "wandb.bucket.secret" . }}"
+                  key: {{ .Values.global.bucket.accessKeyName }}
+                  optional: true
+              {{- end }}
+            - name: SECRET_KEY
+              {{- if .Values.global.bucket.secretKey }}
+              value: "{{ .Values.global.bucket.secretKey | default .Values.global.defaultBucket.secretKey }}"
+              {{- else }}
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ include "wandb.bucket.secret" . }}"
+                  key: {{ .Values.global.bucket.secretAccessKeyName }}
+                  optional: true
+              {{- end }}
```

Comment:
This block can be optional and conditioned to the access/secret pair or secret name.
If the user doesn't pass any of those, we don't need to set these env vars.
The reason is that users can rely on IAM when using AWS.